modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/GroupResourceManager.java

```diff
@@ -932,14 +933,24 @@ private void processValueAttributeOfOperation(SCIMResourceTypeSchema schema, Pat
     }
 
     private List<Map<String, String>> transformMembersAttributeToMap(MultiValuedAttribute multiValuedMembersAttribute)
-            throws CharonException {
+            throws CharonException, BadRequestException {
 
         List<Map<String, String>> memberList = new ArrayList<>();
         List<Attribute> subValuesList = multiValuedMembersAttribute.getAttributeValues();
         for (Attribute subValue : subValuesList) {
             ComplexAttribute complexAttribute = (ComplexAttribute) subValue;
             Map<String, Attribute> subAttributesList = complexAttribute.getSubAttributesList();
 
+            if (!subAttributesList.containsKey(SCIMConstants.CommonSchemaConstants.VALUE) ||
```

Comment:
```
private void throwBadRequestException(String description) {
    throw new BadRequestException(ResponseCodeConstants.DESC_BAD_REQUEST, description);
}

// ... Your main code ...

if (!subAttributesList.containsKey(SCIMConstants.CommonSchemaConstants.VALUE) ||
    !subAttributesList.containsKey(SCIMConstants.CommonSchemaConstants.DISPLAY)) {
    throwBadRequestException(ResponseCodeConstants.INVALID_SYNTAX);
}

else if (StringUtils.isEmpty(((SimpleAttribute) subAttributesList.get(SCIMConstants.CommonSchemaConstants.VALUE)).getStringValue())) {
    throwBadRequestException(ResponseCodeConstants.INVALID_VALUE);
}
```

Reply:
Shall I use the following constructor, it is already implemented.
```
public BadRequestException(String scimType) {
        this(ResponseCodeConstants.DESC_BAD_REQUEST, scimType);
    }
 ```